.flake8

```diff
@@ -0,0 +1,2 @@
+[flake8]
+max-line-length = 88
```

Comment:
Oh ok. But as you can see in #45 I never mentioned black to be used. I'm open to a discussion if we should add `black` as a `pre-commit` hook but for now because of probably removing `black` out of the workflow when using pre-commit hooks, I think it wouldn't make sense to introduce any `black` specific configuration.

Reply:
I know you haven't but `black` made `flake8` happy in just few seconds. But I don't insist on using this, neither here, nor in the pre-commit hooks stuff.